demo/demo_vgg16.ipynb

```diff
@@ -24,33 +27,66 @@
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "INFO:tensorflow:Restoring parameters from ../models/vgg_16/vgg16.ckpt\n",
+      "Activation Not Found.\n"
+     ]
+    }
+   ],
    "source": [
     "#initializing the model\n",
     "\n",
-    "# Specifying to use the GPU\n",
-    "caffe.set_mode_gpu()\n",
-    "caffe.set_device(0)\n",
+    "if BACKEND == 'tf':\n",
+    "    model_path = os.path.join('../models', 'vgg_16', 'vgg16.ckpt.data-00000-of-00001')\n",
+    "    index_path = os.path.join('../models', 'vgg_16', 'vgg16.ckpt.index')\n",
+    "    graph_path = os.path.join('../models', 'vgg_16', 'vgg16.ckpt.meta')\n",
+    "    checkpoint_path = os.path.join('../models', 'vgg_16', 'checkpoint')\n",
+    "    if not os.path.isfile(model_path):\n",
+    "        print('Downloading VGG-16 weights...')\n",
+    "        urlretrieve ('https://drive.google.com/file/d/0B7_JBKQvp2qbUXhpZWl6eTd4eEk/view?usp=sharing', model_path)\n",
+    "        urlretrieve ('https://drive.google.com/file/d/0B7_JBKQvp2qbTU9JbW82OWJhMEk/view?usp=sharing', checkpoint_path)\n",
+    "        urlretrieve ('https://drive.google.com/file/d/0B7_JBKQvp2qbS1JZMDdyNkNZODQ/view?usp=sharing', graph_path)\n",
+    "        urlretrieve ('https://drive.google.com/file/d/0B7_JBKQvp2qbbFZ4SHprSGtESDA/view?usp=sharing', index_path)\n",
```

Comment:
These do not work ( I don't think you can get stuff from google drive using `urlretrieve` )